### DISCUSSION TITLE
Вам делали соскоп?

### DISCUSSION THREAD

**Discussion by katikn** [D1ffic00lt/DPCoinBot#68] [Q&A] [answered]
Вы когда-нибудь задумывались, что в детстве вам делали соскоп в жопу. Вот я недавно задумался, и мне стало интересно, можно ли сделать его завтра, чтобы не пойти на урок мартыновой? Жду ваши answers фолловеры ;)

**Comment by D1ffic00lt** [comment, marked as answer]
новая функция "соскоп" будет в обновлении 2.9.11

**Reply by katikn** [reply]
Спасибо большое!!!! Очень быстро отвечаете обажааю дп соин бота ЛУЧШИЙ бот на свете красава d1ffic00lt ваще молодчик
